Building-with-IntelliJ-IDEA.md

```diff
@@ -20,29 +22,24 @@ You can build RuneLite locally using [JDK 11](https://adoptium.net/temurin/relea
 
 ## Importing the project
 
-After launching IntelliJ IDEA for first time, you will see IntelliJ welcome window. You will now need to clone RuneLite repository from git:
+After launching IntelliJ IDEA for first time, you will see the IntelliJ welcome window. You will now need to clone RuneLite repository from git, to do this you can click the `Get from VSC` button.
 
-![welcome-import-git](https://i.imgur.com/bALnvDY.png)
+![get_from_vcs.png](img%2Fbuilding-with-intellij%2Fget_from_vcs.png)
```

Comment:
Could you replace all the ``%2F`` with slashes everywhere as mentioned here?
So this should be ``![get_from_vcs.png](img/building-with-intellij/get_from_vcs.png)`` instead of ``![get_from_vcs.png](img%2Fbuilding-with-intellij%2Fget_from_vcs.png)``